components/BottomNavigation/src/MDCBottomNavigationBarController.h

```diff
@@ -0,0 +1,68 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class MDCBottomNavigationBar;
+@protocol MDCBottomNavigationBarDelegate;
+@protocol MDCBottomNavigationBarControllerDelegate;
+
+/**
+ * MDCBottomNavigationBarController is a class that manages the navigation bar that allows movement
+ * between primary destination in an app.  It ties a list of view controllers to the bottom
+ * navigation bar and will display the corresponding view controller in the content view when an
+ * in the navigation bar is selected.
+ */
+@interface MDCBottomNavigationBarController : UIViewController <MDCBottomNavigationBarDelegate>
```

Comment:
Can we make the conformance to MDCBottomNavigationBarDelegate a private implementation detail?

Reply:
The intent behind conforming to MDCBottomNavigationBarDelegate protocol in the header file is that it provides transparency to an class that might subclass it and prevents any accidental overriding.

https://github.com/stephanemoore/archives/blob/master/objc/pitfalls/private-protocol-conformances.md